Add evernote extension

artpi opened the pull request:
## Description

This searches Evernote local database used by the official Evernote client for instant search

## Screencast

https://github.com/user-attachments/assets/d79ea308-e0d5-4b89-8a22-d99484b6b7bb



## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

pernielsentikaer reviewed (changes requested):
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Add an extension icon, you should not use the placeholder image 🙂

- When evernote is not found it would be better to handle it a bit like how [Eagle](https://www.raycast.com/Yukai/eagle) is doing it, could you improve this?

![evernote 2024-10-22 at 10 43 54](https://github.com/user-attachments/assets/2ac22f3b-66e7-4779-949b-3ca334d9b668)

- Could you add at least one [screenshot](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) of the extension

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

artpi commented:
Thank you @pernielsentikaer !

I 
- Added an icon (duh, forgot to do that before)
- Added a toast-based error feedback mechanism
- Added a screenshot. Is it enough just to add it in the metadata folder, or is there more needed? Couldnt find any information and the capture metadata tool is not working for me for some reason.

pernielsentikaer reviewed (approved):
Let's get the first version out so we can iterate on it
